google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -47,7 +47,8 @@ appengine.cloudsdk.java.components.howtoinstall=To install, run "gcloud componen
 appengine.cloudsdk.location.label=Cloud SDK directory\:
 appengine.cloudsdk.location.browse.window.title=Browse to Cloud SDK directory
 appengine.cloudsdk.location.badchars.message=The specified Cloud SDK directory contains invalid characters or starts or ends with spaces.
-appengine.cloudsdk.location.invalid.message=No Cloud SDK was found in this directory.
+appengine.cloudsdk.location.invalid.message=No Cloud SDK was found in the specified directory.
+appengine.cloudsdk.location.invalid.message.deploy=The Cloud SDK is misconfigured. To fix, reconfigure the Server above.
```

Comment:
I still don't want us to put UI positional references in our messages. It's  aUI anti-pattern.

appengine.deploymentconfiguration.cloudsdk.location.invalid.message?